lib/components/term.js

```diff
@@ -98,9 +106,24 @@ export default class Term extends Component {
 
     this.getScreenNode().addEventListener('mouseup', this.handleMouseUp);
     this.getScreenNode().addEventListener('mousedown', this.handleMouseDown);
+    this.getScreenNode().addEventListener('contextmenu', this.handleRightClick);
     terms[this.props.uid] = this;
   }
 
+  handleRightClick(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    const {quickEdit} = config.getConfig();
+    const show = !quickEdit;
+
+    const contextMenu = Menu.buildFromTemplate(contextMenuTemplate);
+    if (show) {
+      // throttle popup so right click has time to highlight text
+      setTimeout(() => contextMenu.popup(getCurrentWindow()), 100);
```

Comment:
@chabou I've made some changes that add a redux action + state/reduce

it's currently toggling the contextMenu as an `effect` in the actions, which I don't think is quite right :thinking:

> make a rpc call and popup context menu from app

could you maybe elaborate on that, where _ideally_ would you like to see this/what are your thoughts on setting that up? (sorry this code base is still new to me) 😅